File: visits-service-new/src/main/java/com/petclinic/visits/visitsservicenew/DataLayer/Visit.java
package com.petclinic.visits.visitsservicenew.DataLayer;


import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
import lombok.*;
import org.springframework.data.annotation.Id;


import java.time.LocalDateTime;

/**
 * Visit Entity
 */
@Builder
@Data
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class Visit {

    @Id
    private String id;

    private String visitId;

    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
    LocalDateTime visitDate;

    private String description;

    private String petId;

    private String practitionerId;

    private Status status;

    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
    LocalDateTime visitEndDate;


}

File: emailing-service/emailing-service/Models/Visits/Visit.cs
namespace emailing_service.Models.Visits
{
    using System;

    public class Visit
    {
        public Visit() { }
        public string Id { get; set; }
        public string VisitId { get; set; }
        public DateTime VisitDate { get; set; }
        public string Description { get; set; }
        public string PetId { get; set; }
        public string PractitionerId { get; set; }
        public Status Status { get; set; }
        public DateTime VisitEndDate { get; set; }
        public bool Reminder { get; set; }
        public string OwnerEmail { get; set; }
    }
}
